core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -130,7 +130,7 @@ class JsonProtocolSuite extends SparkFunSuite {
     testEvent(executorUnblacklisted, executorUnblacklistedJsonString)
     testEvent(nodeBlacklisted, nodeBlacklistedJsonString)
     testEvent(nodeUnblacklisted, nodeUnblacklistedJsonString)
-    testEvent(executorMetricsUpdate, executorMetricsUpdateJsonString)
+//    testEvent(executorMetricsUpdate, executorMetricsUpdateJsonString)
```

Comment:
looks like you fixed the expectation below, any reason its still commented out?

Reply:
My typo. Updated